lib/rialto/etl/service_client/entity_resolver.rb

```diff
@@ -22,7 +22,13 @@ def initialize
 
         def resolve(type, params)
           resp = conn.get(type, params)
-          return resp.body if resp.success?
+          if resp.success?
+            return RDF::URI.new(resp.body) if resp.success?
```

Comment:
I think the `if` statement on line 26 is not needed thanks to the `if` statement on the prior line.

Reply:
Apparently, I really wanted to be extra sure that the response was a success.